README.md

```diff
@@ -49,3 +49,33 @@ gulp serve
 Serve task will create a proxy server on port 3000 that goes via Django’s server on port 8000 enhancing it with
 additional features provided by [Browsersync](http://www.browsersync.io/), such as CSS/JS reload, interaction
 synchronisation and much more.
+
+## Releasing
+
+### Releasing to non-production
+
+> Currently, `staging` is the only non-production environment.
+
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/fala) for the feature branch.
+1. Copy the `feature_branch.<sha>` reference from the `build` job's "Push Docker image" step. Eg:
+    ```
+    Pushing tag for rev [3e1a4ef9ecf6] on {https://registry.service.dsd.io/v1/repositories/fala/tags/github-config.843dcc7}
+    ```
+1. [Deploy `feature_branch.<sha>`](https://ci.service.dsd.io/job/DEPLOY-fala/build?delay=0sec).
+    * `ENVIRONMENT` is the target environment, select "staging".
+    * `DEPLOY_BRANCH` is the [deploy repo's](https://github.com/ministryofjustice/fala-deploy) default branch name, usually `master`.
+    * `VERSION` is the branch that needs to be released plus a specific 7-character prefix of the Git SHA. (`github-config.843dcc7` for the above example).
+
+
+### Releasing to production
+
+1. Please make sure you tested on a non-production environment before merging.
+1. Merge your feature branch pull request to `master`.
```

Comment:
🙋‍♂️ This repository does not use `master` as the default branch. Should we switch?

Reply:
Yeap, that's why I wrote:

>We will switch the default branch on this repository from develop to master once this is merged.

in the pull request description.

We should do a release as soon as we merge develop into master, then set `master` as default branch.

Reply:
Right, I missed that, sorry :(